### DISCUSSION TITLE
Split sensitivity for gameplay and menus

### DISCUSSION THREAD

**Discussion by HoutarouOreki** [ppy/osu#13171] [Ideas]
I use 4.25x sensitivity with my tablet and I'd prefer using my mouse for menus. However my mouse-cursor is too sensitive now so I'm forced to use the tablet. It could be nice to have a different sensitivity while in menus.

**Comment by bdach** [comment]
I find that the menus are pretty stylus-usable as they are, but that's just my opinion.

**Comment by peppy** [comment]
Sensitivity should really be per input device.

**Comment by M4xi1m3** [comment]
I'll allow myself to up this thread since the game has changed since then. I think both the per-device sensitivity + game/menu sensitivity should be there. I've been starting to play with a tablet, and I navigate in the menu with it, but my play area is set to be really small, and navigating in the menus like that is very very hard, so I think both should be done (or have a default for the menus, like the full area of the tablet, or an option to enable that).

**Comment by Scrollname** [comment]
I also would like to have separate sensitivity for gameplay and menu, as I play on high sensitivity and there are very small buttons in menu so it's hard to aim at them when you have very high sensitivity. Maybe some kind of button to use regular mouse sensitivity in menu and set sensitivity while playing.

**Reply by peppy** [reply]
Most mice have the ability to adjust DOI using hardware buttons. Maybe you can try that?

**Reply by Scrollname** [reply]
You mean changing DPI when I'm in menu? Well I could do that, but my DPI switching button is under my mouse and it take around 10 presses to go on lower and back. The thing is that mouse sensitivity in menu and in game are same slider. Also I had a friend who wanted to change his sensitivity and couldn't choose exactly because he had high sensitivity set already, so his cursor was too fast.

**Comment by jarvis394** [comment]
I too have a pretty high sensitivity on a tablet but I don't remember clicking menus in stable to be a problem. When I switched to lazer however, I found myself missing menu items or not being able to open menus from first click at all. Had a look on this problem in  https://github.com/ppy/osu-framework/discussions/6584 and did a fix in https://github.com/ppy/osu-framework/pull/6610

The general problem is that some clickables that were being clicked by just pressing on the tablet in stable, now only clicked with press and release in lazer. I guess stable does what it does to mitigate shaky cursor presses for tablets. The PR above fixes behavior for menus and for menu items the fix can be a bit trickier (in stable they are also sometimes clicked by just pressing).

**Reply by pigeonskew** [reply]
Pretty sure it's harder to press menus on lazer since they're way smaller to press than on stable.
